### PR TITLE
VACMS-000: Add label to covid fieldset.

### LINKED ISSUE #9064: Add fieldgroup label to covid status
## Description
The field group for node view on vamc facility  for supplemental status should say "COVID-19 guidelines"

## Acceptance Criteria
- [ ] Testable_Outcome_X
- [ ] Testable_Outcome_Y
- [ ] Testable_Outcome_Z
- [ ] Requires design review

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by ethanteague**
## Description

- Adds fieldset label on facility node view to covid status 
- Expands link on help text on covid field on help form / moves period after parenthesis
- Updates form operating status help text

closes #9064 

## Testing done
Visual

## Screenshots
image

image

image

## QA steps
 - [ ] As an administrator, go to `pittsburgh-health-care/locations/belmont-county-va-clinic` and visually verify Covid status fieldset has `COVID-19 guidelines` label
 - [ ] Go to `node/90/edit` and visually verify `COVID-19 status` field help text link extends to end of parenthesis and period appears after.
 - [ ] On `node/90/edit` visually verify Operating status fieldset helptext has `Don’t use this for COVID-19-related information.` replaced with `Don’t duplicate information contained in your COVID-19 status.`

### CODE REVIEW

File: config/sync/field.field.node.health_care_local_facility.field_supplemental_status.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ field_name: field_supplemental_status
 entity_type: node
 bundle: health_care_local_facility
 label: 'COVID-19 status'
-description: 'Select the level set by your facility’s leadership. Learn more about COVID-19 safety guidelines <a href="/help/vamc/about-locations-content-for-vamcs/about-alerts-and-operating-statuses/how-to-edit-a-vamc-facility-covid-status" target="_blank">Learn more about COVID-19 safety guidelines</a>.(opens in a new window)'
+description: 'Select the level set by your facility’s leadership. Learn more about COVID-19 safety guidelines <a href="/help/vamc/about-locations-content-for-vamcs/about-alerts-and-operating-statuses/how-to-edit-a-vamc-facility-covid-status" target="_blank">Learn more about COVID-19 safety guidelines(opens in a new window)</a>.'
```
Comment (swirtSJW) [inline config/sync/field.field.node.health_care_local_facility.field_supplemental_status.yml:24]:
```suggestion
description: 'Select the level set by your facility’s leadership. Learn more about COVID-19 safety guidelines <a href="/help/vamc/about-locations-content-for-vamcs/about-alerts-and-operating-statuses/how-to-edit-a-vamc-facility-covid-status" target="_blank">Learn more about COVID-19 safety guidelines (opens in a new window)</a>.'
```

**Review by swirtSJW — APPROVED** [review]
Approved, but dont merge yet.  I'll merge them back to back

**Comment by ethanteague** [comment]
> Approved, but dont merge yet. I'll merge them back to back

Thanks & thanks for taking care of that spacing issue, @swirtSJW

**Comment by swirtSJW** [comment]
Closing this as I cherry-picked the commits into integration-2

**Pull request closed without merging**